include/cuco/static_multimap.cuh

```diff
@@ -201,6 +200,25 @@ class static_multimap {
                   cudaStream_t stream    = 0,
                   Allocator const& alloc = Allocator{});
 
+  /**
+   * @brief Move-constructor
+   * @param other Object to be moved
+   */
+  static_multimap(static_multimap&& other)
```

Comment:
I don't see any data members that aren't default moveable. What prevents us from defaulting this ctor?

Reply:
I've replaced it with the default one.